services/QuillLMS/spec/workers/admin_dashboard/admin_account_created_email_worker_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# perform(user_id, admin_user_id, school_id, is_reminder)
+
+describe AdminDashboard::AdminAccountCreatedEmailWorker, type: :worker do
+  let!(:teacher) { create(:teacher) }
```

Comment:
Instead of the comment on line 5, use a 
`subject { perform(user_id, admin_user_id, school_id, is_reminder)`

on line 8.

This does two things:
1) it standardizes what will be tested in each spec
2) if the args for the worker change, the spec should fail, whereas the comment might go stale.

Then replaces spec with `subject`, for example:

```
  it 'should not send the mail with user mailer' do
      expect(mailer_class).not_to receive(mailer_method)
      subject
    end
```